<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Client Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #333333;
}

QLineEdit:focus {
    border-color: #0078d7;
    outline: none;
}

QLineEdit::placeholder {
    color: #999999;
    font-style: italic;
}

/* For invalid input state */
QLineEdit[error=&quot;true&quot;] {
    border-color: #d9534f;
}

QFrame{

	background-color:#f4f5fc;


}
QWidget#Form{
background-color:#f4f5fc;


}

QComboBox {
    border: 1px solid #cccccc;
    border-radius: 5px;
    padding: 5px;
    background-color: #ffffff;
    color: #333333;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 14px;
}


QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: #cccccc;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/icons/down-arrow.png); /* Ensure you have an appropriate icon */
}

QComboBox QAbstractItemView {
    border: 1px solid #cccccc;
    border-radius: 5px;
    background-color: #ffffff;
    selection-background-color: #24293e;
    selection-color: #ffffff;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 14px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>401</width>
       <height>701</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>401</width>
       <height>701</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 14px;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    padding: 5px;
    background: #24293e;
    border: 1px solid #dddddd;
    border-radius: 5px;
    text-align: center;
    qproperty-alignment: 'AlignCenter';
}

QLabel[warning=&quot;true&quot;] {
    color: #d9534f;
    background: #f2dede;
    border: 1px solid #ebccd1;
}

QLabel[success=&quot;true&quot;] {
    color: #3c763d;
    background: #dff0d8;
    border: 1px solid #d6e9c6;
}

QLabel[info=&quot;true&quot;] {
    color: #31708f;
    background: #d9edf7;
    border: 1px solid #bce8f1;
}
</string>
        </property>
        <property name="text">
         <string>CLIENT DETAILS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contact">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Contact Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Date of Pickup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Time of Pickup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pax">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Number of Clients(PAX)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="location">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Location of Pickup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeCbox">
        <property name="styleSheet">
         <string notr="true">  QComboBox {
                padding-left: 10px;
            }
 QComboBox QAbstractItemView {
                text-align: center;
            }</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destination">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cost">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Cost</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="savebutton">
          <property name="styleSheet">
           <string notr="true">QPushButton{

border-radius: 25px;
color: white;
background-color: #24293e;
padding-top: 20px;
padding-bottom: 20px;






}</string>
          </property>
          <property name="text">
           <string>SAVE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelbutton">
          <property name="styleSheet">
           <string notr="true">QPushButton{

border-radius: 25px;
color: white;
background-color: #24293e;
padding-top: 20px;
padding-bottom: 20px;






}</string>
          </property>
          <property name="text">
           <string>CANCEL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
